docs/contents/components/data-display/carousel/props.mdx

```diff
@@ -67,7 +67,7 @@ tab: Props
 <PropsCard
   id="carousel-controlref"
   name="controlRef"
-  type="RefObject<EmblaCarouselType | undefined>"
+  type="RefObject<CarouselControl | undefined>"
```

Comment:
This file is automatically generated and does not need to be changed.

Reply:
I understand!
I'll fix it!